src/PSR7Client.php

```diff
@@ -144,8 +172,12 @@ private function wrapUploads($files): array
                 continue;
             }
 
-            $result[$index] = new Diactoros\UploadedFile(
-                $f['tmpName'],
+            $stream = UPLOAD_ERR_OK === $f['error'] ?
```

Comment:
Let's stick to if/else there. Ternary operator does not make it easier to understand :)

Reply:
Sure, no problem.